chia/consensus/coinbase.py

```diff
@@ -7,18 +7,7 @@
 
 
 def create_puzzlehash_for_pk(pub_key: G1Element) -> bytes32:
-    return puzzle_for_pk(bytes(pub_key)).get_tree_hash()
```

Comment:
This bytes() was unnecessary, since this method takes in a G1Element, and later calls bytes() on it